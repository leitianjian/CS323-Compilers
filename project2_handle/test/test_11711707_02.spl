int test(int c){
    int cc = c, d, e[1];
    float c;
    c = 3.0;
    d = 1;
    e[0] = 0;
    return 1;
}