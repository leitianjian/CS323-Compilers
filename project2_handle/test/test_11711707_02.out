Error type 3 at Line 3: c variable is redefined in the same scope
